docs/administration/configuration.md

```diff
@@ -1684,15 +1684,28 @@ This given password is then encrypted **by the same algorithm and the same encod
 
 As of eXo Platform 4.3, the encoder and the algorithm can be configured via `exo.properties` file.
 
-::: tip
-
-It is not likely administrators will want to change the default encoder and algorithm. However for users who upgrade from a previous version older than 4.3, it is important to know that **the default encoder and the default algorithm have changed**, so you will need to re-configure it back to the old one which has been used, otherwise old users will not be able to log in.
-:::
+As of eXo Platform 6.5, the default encoder change to use a more robust one. Before change, default encoder was `org.picketlink.idm.impl.credential.DatabaseReadingSaltEncoder` with SHA-256 hashing algorithm.
+Now, the default encoder is `org.exoplatform.web.security.hash.Argon2IdPasswordEncoder`.
 
   Name | Description | Default
   -----|-------------|------------
-  exo.plidm.password.class|The class that encrypts the user password before it is stored in the database. | org.picketlink.idm.impl.credential.HashingEncoder
-  exo.plidm.password.hash | The encryption algorithm | SHA-256
+  exo.plidm.password.class|The class that encrypts the user password before it is stored in the database. | org.exoplatform.web.security.hash.Argon2IdPasswordEncoder
+
+### Update Password Encryption Algorithm
+
+To make this important change transparent for administrators who are in charge of platform security, there is few points to know : 
+
+- It is possible to keep the algorithm you use previously, by setting the property with the encoder class you want. As the DatabaseReadingSaltEncoder is less secure, keeping the old encoder could lead to security vulnerabilities
+- It is possible to move to the new hashing algorithm. For that, unset the `exo.plidm.password.class` (default class will be used aka Argon2IdPasswordEncoder), and set the property `exo.plidm.password.oldClass` with the previous encoder class. 
+
```

Comment:
Is the exo.plidm.password.oldClass needed for all existant instances with old algorithm or just in case We need to keep it?

Reply:
This property will be necessary to all existant instance who want to update the algo. 

If you have an existant instance who dont want to update the algo, only keep exo.plidm.password.class
If you have a new instance, no property needed.
The property oldClass is needed as longer as there is at least a user with password encoded with old algo not reconnected since the algo switch. As we are not able to know there is such user, I'm afraid that this property is needed for all lifetime of existing instance

Reply:
ok, with the exo.plidm.password.oldClass should we give the exo.plidm.password.hash as for some instances like tribe the hash is not sha-256 but md5

Reply:
yes, no changes for hash property. In fact, hash property, is used by encoder class, and the new encoder Argon2 doesnt need hash algo property.
So hash property must be kept in order DatabaseReadingEncoder of HashingEncoder continue to work